.github/workflows/_rust.yml

```diff
@@ -1,11 +1,15 @@
+---
 name: Rust
-on:
+"on":
   workflow_call:
 
 defaults:
   run:
     working-directory: ./rust
 
+macos-packages: &macos-packages "-p connlib-client-apple -p connlib-client-shared -p firezone-tunnel -p snownet"
```

Comment:
I added a couple packages that should be able to build and test on macOS, since they're built for prod anyway.